core/src/Cabana_CommunicationPlan.hpp

```diff
@@ -798,18 +798,23 @@ class CommunicationPlan
         _export_steering = Kokkos::View<std::size_t*, memory_space>(
             Kokkos::ViewAllocateWithoutInitializing( "export_steering" ),
             _total_num_export );
-        auto steer_vec = _export_steering;
-        Kokkos::View<std::size_t*, memory_space> counts( "counts", num_n );
-        Kokkos::parallel_for(
-            "Cabana::createSteering",
-            Kokkos::RangePolicy<execution_space>( 0, _num_export_element ),
-            KOKKOS_LAMBDA( const int i ) {
-                if ( element_export_ranks( i ) >= 0 )
-                    steer_vec( rank_offsets( element_export_ranks( i ) ) +
-                               neighbor_ids( i ) ) =
-                        ( use_iota ) ? i : element_export_ids( i );
-            } );
-        Kokkos::fence();
+        // It is possible to have export elements, but not any to export (e.g.
+        // all elements to be removed).
+        if ( _total_num_export > 0 )
+        {
+            auto steer_vec = _export_steering;
+            Kokkos::View<std::size_t*, memory_space> counts( "counts", num_n );
```

Comment:
I see that it was it before but why do we have this array in the first place?  It is not used.  Also how comes we don't get a warning about it?

Reply:
Yeah, I'm really not sure why - I removed it